xcm/Cargo.toml

```diff
@@ -13,7 +13,7 @@ log = { version = "0.4.17", default-features = false }
 parity-scale-codec = { version = "3.4.0", default-features = false, features = [ "derive", "max-encoded-len" ] }
 scale-info = { version = "2.5.0", default-features = false, features = ["derive"] }
 sp-weights = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
-serde = { version = "1.0.136", optional = true, features = ["derive"] }
+serde = { version = "1.0.136", optional = true, features = ["derive", "rc"] }
```

Comment:
Will we need to do similar for subxt?

Reply:
Ah I see https://github.com/paritytech/parity-scale-codec/pull/426 is related.

Reply:
I see you've laid the groundwork for this: https://github.com/paritytech/parity-scale-codec/pull/439 and https://github.com/paritytech/scale-info/pull/180 . As long as subxt is using the latest libs then this should be fine.